public_html/wp-content/mu-plugins/events/redirects.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace WordCamp\Redirects;
+
+defined( 'WPINC' ) || die();
+
+add_action( 'init', __NAMESPACE__ . '\redirect_to_make_community' );
+
+/**
+ * Redirects the root URL of the "events.wordcamp.test" to "https://make.wordpress.org/community/events".
```

Comment:
```suggestion
 * Redirects the root URL of the "events.wordcamp.org" to "https://make.wordpress.org/community/events".
```
Should this be .org? At first glance this appeared to be a patch for testing purposes but I don't think that's the case correct?

Reply:
Good catch, missed that 👍